### PR TITLE
fix: integrations bugsnag alerts

### PR DESCRIPTION
**Pull request by mihir-4116**
## PR Description

Please include a summary of the change along with the relevant motivation and context.

## Linear task (optional)

Linear task link

## Cross Browser Tests

Please confirm you have tested for the following browsers:

- [ ] Chrome
- [ ] Firefox
- [ ] IE11

## Sanity Suite

- [ ] All sanity suite test cases pass locally

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: packages/analytics-js-integrations/src/integrations/LinkedInInsightTag/browser.js [hunk 1]
```diff
@@ -49,6 +49,12 @@ class LinkedInInsightTag {
       logger.error('Event name is missing for track call');
       return;
     }
+
+    if (typeof event !== 'string') {
+      logger.error('Event type should be string');
```
Comment (saikumarrs) [inline packages/analytics-js-integrations/src/integrations/LinkedInInsightTag/browser.js:54]:
```suggestion
      logger.error('Event name should be a string');
```

File: packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js [hunk 2]
```diff
@@ -44,6 +44,17 @@ class QuoraPixel {
   track(rudderElement) {
     const { event } = rudderElement.message;
     const eventsMapping = getHashFromArrayWithDuplicate(this.eventsToQPEvents);
+
+    if (!event) {
+      logger.error('Event name is missing for track call');
```
Comment (saikumarrs) [inline packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js:49]:
```suggestion
      logger.error('Event name is not defined');
```

File: packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js [hunk 3]
```diff
@@ -44,6 +44,17 @@ class QuoraPixel {
   track(rudderElement) {
     const { event } = rudderElement.message;
     const eventsMapping = getHashFromArrayWithDuplicate(this.eventsToQPEvents);
+
+    if (!event) {
```
Comment (saikumarrs) [inline packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js:48]: Use an utility method to check for undefined and null values.

File: packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js [hunk 4]
```diff
@@ -44,6 +44,17 @@ class QuoraPixel {
...
+      logger.error('Event name is missing for track call');
+      return;
+    }
+
+    if (typeof event !== 'string') {
+      logger.error('Event type should be string');
```
Comment (saikumarrs) [inline packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js:54]:
```suggestion
      logger.error('Event name should be a string');
```

File: packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js (hunk 3 above)
Comment (saikumarrs) [reply packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js:48]: You can reuse the Typescript utilities from here,
@rudderstack/analytics-js-common/utilities/

**Review by saikumarrs — CHANGES_REQUESTED** [review]
Added comments for improvement.

File: packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js (hunk 4 above)
Comment (mihir-4116) [reply packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js]: addressed

File: packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js (hunk 3 above)
Comment (mihir-4116) [reply packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js]: addressed

File: packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js (hunk 2 above)
Comment (mihir-4116) [reply packages/analytics-js-integrations/src/integrations/QuoraPixel/browser.js]: addressed

File: packages/analytics-js-integrations/src/integrations/LinkedInInsightTag/browser.js (hunk 1 above)
Comment (mihir-4116) [reply packages/analytics-js-integrations/src/integrations/LinkedInInsightTag/browser.js]: addressed

File: packages/analytics-js-integrations/src/integrations/LinkedInInsightTag/browser.js [hunk 5]
```diff
@@ -5,6 +5,7 @@ import {
   NAME,
   DISPLAY_NAME,
 } from '@rudderstack/analytics-js-common/constants/integrations/LinkedInInsightTag/constants';
+import { isDefinedAndNotNull } from '@rudderstack/analytics-js-integrations/utils/utils';
```
Comment (saikumarrs) [inline packages/analytics-js-integrations/src/integrations/LinkedInInsightTag/browser.js:8]: Can you use the utilities from here?
@rudderstack/analytics-js-common/utilities/

Comment (mihir-4116) [reply packages/analytics-js-integrations/src/integrations/LinkedInInsightTag/browser.js]: done

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**